origami.json

```diff
@@ -19,12 +19,14 @@
 		{
 			"name": "palette",
 			"template": "demos/src/palette.mustache",
-			"description": ""
+			"description": "",
+			"display_html": "false"
```

Comment:
erm, only because I copied the line above and forgot that boolean's don't need `""`'s... And it's Monday. 

Will fix now.